schemas/event.yml

```diff
@@ -38,53 +38,174 @@
+            relational database systems. Examples include event logs from MS SQL, MySQL,
+            Elasticsearch, MongoDB, etc. Use this category to visualize and analyze
+            database activity such as accesses and changes.
+          expected_event_types:
+            - access
+            - change
+            - info
+            - error
+        - name: driver
+          description: >
+            Having to do operating system device drivers and similar software entities
+            such as Windows drivers, kernel extensions, kernel modules, etc.
+            Use events and metrics in this category to visualize and analyze driver-related
+            activity and status on hosts.
+          expected_event_types:
+            - change
+            - end
+            - info
+            - start
+        - name: file
+          description: >
+            Relating to a set of information that has been created on, or has existed on a filesystem.
+            Use this category of events to visualize and analyze the creation, access,
+            and deletions of files. Events in this category can come
+            from both host-based and network-based sources. An example source of a network-based
+            detection of a file transfer would be the Zeek file.log.
+          expected_event_types:
+            - change
+            - creation
+            - deletion
+            - info
+        - name: host
+          description: >
+            Events and metrics about hosts. Usually higher-level information about host
+            activity from an external perspective. Different than operating system
+            in the sense that host events are usually externally visible and independent from the OS.
+            Note that `event.category:host` is not meant to indicate events that are simply
+            "happening on a host". Use this category to visualize and analyze inventories
+            of hosts, starting and ending of hosts, etc.
+          expected_event_types:
+            - access
+            - change
+            - end
+            - info
+            - start
+        - name: intrusion_detection
+          description: >
+            Relating to intrusion detections from IDS/IPS systems and functions,
+            both network and host-based. Use this category to visualize and analyze
+            intrusion detection alerts from systems such as Snort, Suricata,
+            and Palo Alto threat detections.
+          expected_event_types:
+            - info
+        - name: malware
+          description: >
+            Malware detection events and alerts. Use this category to visualize and analyze
+            malware detections from EDR/EPP systems such as Elastic Endpoint Security,
+            Symantec Endpoint Protection, Crowdstrike, and network IDS/IPS systems and
+            functions such as Palo Alto Networks threat and Wildfire logs.
```

Comment:
other sources of malware-related events such as Palo Alto Networks threat logs and Wildfire logs.